module/applications/activity/activity-sheet.mjs

```diff
@@ -435,6 +474,23 @@ export default class ActivitySheet extends Application5e {
 
   /* -------------------------------------------- */
 
+  /**
+   * Handle deleting an active effect and removing it from the applied effects list.
+   * @this {ActivityConfig}
+   * @param {Event} event         Triggering click event.
+   * @param {HTMLElement} target  Button that was clicked.
+   */
+  static async #deleteEffect(event, target) {
+    const effectId = target.closest("[data-effect-id]")?.dataset.effectId;
+    const result = await this.item.effects.get(effectId)?.deleteDialog();
+    if ( result !== false ) {
```

Comment:
If the dialog is closed without choosing yes or no, I believe it returns `null`, so this code will proceed in that case.